packages/nuxt/src/pages/module.ts

```diff
@@ -187,6 +187,7 @@ export default defineNuxtModule({
       nuxt.hook('prepare:types', ({ references }) => {
         // This file will be generated by unplugin-vue-router
         references.push({ path: declarationFile })
+        references.push({ types: 'unplugin-vue-router/client' })
```

Comment:
So now, to make things work everywhere, it's better to add a `/// <reference` comment to some `.d.ts` file. It was causing issues with vitest: https://github.com/posva/unplugin-vue-router/issues/353

Reply:
That is what this bit does here. It adds a `/// <reference>` comment.